packages/ui-extensions-go-cli/api/root/templates/checkout_post_purchase/tunnel-error.html.tpl

```diff
@@ -35,7 +35,7 @@
   <body>
   <div class="content">
       <p>This page is served by your local UI Extension development server. Instead of visiting this page directly, you will need to connect your local development environment to a real checkout environment.</p>
-      <p>Make sure you have a secure URL for your local development server by running <code>shopify extension tunnel start --port={{ .Port }}</code>, create a checkout, and append <code>?dev=https://TUNNEL_URL{{.ApiRoot}}/{{.UUID}}</code> to the URL, where <code>TUNNEL_URL</code> is replaced with your own ngrok URL.</p>
+      <p>Make sure you have a secure URL for your local development server by using the <code>--tunnel</code> flag.</p>
```

Comment:
Since now you can potentially execute extensions without a tunnel, this error might appear so I took the opportunity to update the messaging.

Reply:
<img width="818" alt="Captura de Pantalla 2022-08-31 a las 11 36 47" src="https://user-images.githubusercontent.com/3757193/187649694-37bd2d12-7bc4-4bfb-b21f-c7c394da78b7.png">
to
<img width="1975" alt="Captura de Pantalla 2022-08-31 a las 11 36 55" src="https://user-images.githubusercontent.com/3757193/187649698-df76f863-398b-4747-b70f-dedb88fb0631.png">